content/en/docs/v3/getting-started/eks.md

```diff
@@ -20,12 +20,13 @@ This is our current recommended quickstart for EKS:
 
 *  <a href="https://github.com/jx3-gitops-repositories/jx3-terraform-eks/generate" target="github" class="btn bg-primary text-light">Create Git Repository</a>
 
-* `git clone` the new repository and `cd`  into the git clone
+* `git clone` the newly Generated repository and `cd` into it.
```

Comment:
Hi @enkicoma, thank you for your PR! Generated doesn't need to be capitalised and it would be good to have all the getting started guides with similar wording. Changing the git clone line in all the getting-started guides would be great.